ibm/mas_devops/roles/suite_app_install/vars/defaultspecs/predict.yml

```diff
@@ -1,10 +1,7 @@
 ---
 # Default application specs for Predict
 mas_app_spec:
-  bindings:
-    jdbc: "{{ mas_app_bindings_jdbc }}"
   components: {}
   settings:
-    deployment:
-      size: "{{ mas_app_settings_predict_deployment_size}}"
```

Comment:
If we aren't going workload scaling then we still need the deployment size don't we?

Reply:
Hi @whitfiea 
We don't, we disabled this as I implement OM3.1 for MAS 8.10 - but then design has been change. So we don't use this variable any more